lib/avromatic/schema_registry_patch.rb

```diff
@@ -7,24 +7,26 @@ module CacheableSchemaRegistration
     def register(subject, schema)
       return super unless Avromatic.use_cacheable_schema_registration
 
+      begin
+        lookup_subject_schema(subject, schema)
+      rescue
```

Comment:
I intentionally left this rescuing all errors so that it would work even against a server that does not support the endpoint.